### PR TITLE
Update dependencies.

### PR DESCRIPTION
**Pull request by XhmikosR**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -26,17 +26,17 @@
     "test": "grunt test"
   },
   "dependencies": {
-    "clone": "~0.1.11",
-    "parserlib": "~0.2.4"
+    "clone": "~0.2.0",
```
Comment (shinnn) [inline package.json]: v1.0.0 now https://www.npmjs.com/package/clone

File: package.json [hunk 2]
```diff
@@ -26,17 +26,17 @@
...
   "devDependencies": {
-    "grunt": "~0.4.4",
-    "grunt-contrib-clean": "~0.5.0",
-    "grunt-contrib-concat": "~0.4.0",
-    "grunt-contrib-copy": "~0.5.0",
-    "grunt-contrib-jshint": "~0.10.0",
+    "grunt": "~0.4.5",
+    "grunt-contrib-clean": "~0.6.0",
+    "grunt-contrib-concat": "~0.5.0",
+    "grunt-contrib-copy": "~0.7.0",
```
Comment (shinnn) [inline package.json]: 0.7.0 now https://www.npmjs.com/package/grunt-contrib-copy

File: package.json [hunk 3]
```diff
@@ -26,17 +26,17 @@
...
+    "grunt-contrib-jshint": "~0.11.0",
     "grunt-contrib-watch": "~0.6.1",
-    "grunt-include-replace": "~1.2.0",
+    "grunt-include-replace": "~2.0.0",
```
Comment (shinnn) [inline package.json]: 3.0.0 now https://www.npmjs.com/package/grunt-include-replace

Comment (XhmikosR) [inline package.json]: When it's ready to be merged I will update it then.

**Comment by XhmikosR** [comment]
@nschonni: ping

**Pull request merged**